vue/sbc-common-components/package-lock.json

```diff
@@ -1,24 +1,26 @@
 {
   "name": "sbc-common-components",
-  "version": "3.0.16",
+  "version": "3.0.17",
```

Comment:
I thought you were going with 3.1.0 ?

Reply:
for Pinia yup

Reply:
OK, gotcha.

PS Might want to mention in PR title that you're changing to Vite.